RAMP/packer.py

```diff
@@ -91,6 +92,10 @@ def package(module, output, verbose, manifest, display_name, author,
     metadata["semantic_version"] = str(version_to_semantic_version(module.version))
     metadata["commands"] = [cmd.to_dict() for cmd in module.commands]
 
+    if print_only:
+        print output.format(**metadata)
```

Comment:
Isn't output a formatted string, as such you only interested to see how that string is expanded?

Reply:
Exactly